src/NovaDetachedFilters.php

```diff
@@ -13,6 +13,7 @@ class NovaDetachedFilters extends Card
     protected $persistFilters = false;
     protected $perPageOptions = null;
     protected $showPerPageInMenu = true;
+    protected $isPersisting = false;
```

Comment:
Can we rename this to `$isPersitingDefault`  or `persistFiltersDefault`

Reply:
Or something that would better reflect that it's a default value.

Reply:
new commit made to reflect your suggestion